common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -67,13 +67,25 @@ Checking Notch Filter Effectiveness
 
 Once the notch filter(s) are setup, the effectiveness of them can be checked by again measuring the  frequency spectrum of the output of the filters (which are the new inputs to the IMU sensors). Refer back to the :ref:`common-imu-batchsampling`  page for this.
 
-Double/Triple-Notch
-===================
+Multi Notch
+===========
 
-The software notch filters used are very "spikey" being relatively narrow but good at attenuation at their center. On larger copters the noise profile of the motors is quite dirty covering a broader range of frequencies than can be covered by a single notch filter. In order to address this situation it is possible to configure the harmonic notches as double or triple notches that gives a wider spread of significant attenuation. To utilize this feature set :ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` to "1" for double notches, to "16" for triple notches.
+The software notch filters used are very "spikey" being relatively narrow but good at attenuation at their center. On larger copters the noise profile of the motors is quite dirty covering a broader range of frequencies than can be covered by a single notch filter. In order to address this situation it is possible to configure the harmonic notches as multiple notches that gives a wider spread of significant attenuation. The configuration is controlled by the :ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` parameter. This is a bitmask parameter and multiple options are possible at the same time, but using bit 0, 1, or bit 4 at the same time should be avoided.
+
+==========================================      =======================
+:ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` Bit      Action
+==========================================      =======================
+0                                               Double side by side Notches
```

Comment:
Actually both double and triple notches are not side-by-side, but overlapping. Overlapping means that there is still significant attenuation between the notches

Reply:
changed